test/karma-system-config.js

```diff
@@ -112,6 +112,8 @@ System.config({
     '@angular/material/core': 'dist/packages/material/core/index.js',
     '@angular/material/datepicker': 'dist/packages/material/datepicker/index.js',
     '@angular/material/dialog': 'dist/packages/material/dialog/index.js',
+    '@angular/material/dialog/testing': 'dist/packages/material/dialog/testing/index.js',
+    '@angular/material/dialog/testing/shared.spec': 'dist/packages/material/dialog/testing/shared.spec.js',
```

Comment:
Isn't it a little weird to import one test file into another? Instead we could do like we did for the component tests and copy them over. Compared to the component tests, the harness ones are fairly simple.

Reply:
For the components tests, there is a slight amount of variance between implementations. For these tests, though, we super want them to be exactly the same because it's part of our migration strategy (get people to use the harnesses, and then change the underlying implementation).